### ISSUE TITLE
[Bug]- WARNING: [15:59:22][Backup-DbaDatabase] Backup Failed | Cannot bind argument to parameter 'SqlInstance' because it is null. 

### ISSUE THREAD

**Issue by mganci350** [sqlcollaborative/dbatools#6405]
### Environmental information

```powershell
PS C:\Windows\system32> Get-Culture

LCID             Name             DisplayName                                                                                                                                
----             ----             -----------                                                                                                                                
1033             en-US            English (United States
```

#### SQL Server: 

```sql
Microsoft SQL Server 2012 (SP4-GDR) (KB4057116) - 11.0.7462.6 (X64) 
	Jan  5 2018 22:11:56 
	Copyright (c) Microsoft Corporation
	Enterprise Edition: Core-based Licensing (64-bit) on Windows NT 6.3 <X64> (Build 14393: )

```

```sql
/* REPLACE WITH output of @@LANGUAGE */
```

#### Host used

- [ X] powershell.exe
- [ X] ISE
- [ ] VS Code
- [ ] Other (please specify)

If anything other than powershell.exe was used, please confirm that you can duplicate the issue with powershell.exe

- [ ] Still buggy in powershell.exe

#### Actual Behavior

Thanks for your help!
Mark G

**Comment by mganci350** [comment]
One other thing I wanted to assert , the backup files are completed successfully and are good backups. They restore just fine.
thx

**Comment by mganci350** [comment]
Hello,
I can still recreate/reproduce  this issue on        1.0.108    dbatools
Windows OS is Server 2016.  SQL Server is 2012 Developer. 
I run the following command from Remote server with Windows 2016 OS.
Backup-DbaDatabase -SqlInstance sqlinstance\sql_instance -Database dbname123 -Path G:\backupsdir -FileCount 8 -Type Full -ReplaceInName -IgnoreFileChecks

The target SLQ Server is a traditional SQL failover cluster instance and has AG set up.
Target database for backup is Primary replica.
There is no Listener in this AG.

Output:
WARNING: [13:18:13][Backup-DbaDatabase] Backup Failed | Cannot bind argument to parameter 'SqlInstance' because it is null.

Backup is produced successfully.

Thanks

**Comment by andreasjordan** [comment]
This is probably fixed with #6621 - @mganci350 can you test with latest version of dbatools?
